partner_identification/tests/test_res_partner.py

```diff
@@ -2,77 +2,33 @@
 # Copyright 2017 LasLabs Inc.
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
-from odoo import fields, models
 from odoo.tests import common
 from odoo.exceptions import ValidationError
 
 
-class ResPartner(models.Model):
-    _inherit = 'res.partner'
-
-    social_security = fields.Char(
-        compute=lambda s: s._compute_identification(
-            'social_security', 'SSN',
-        ),
-        inverse=lambda s: s._inverse_identification(
-            'social_security', 'SSN',
-        ),
-        search=lambda s, *a: s._search_identification(
-            'SSN', *a
-        ),
-    )
-
-
 class TestResPartner(common.SavepointCase):
 
-    @classmethod
-    def _init_test_model(cls, model_cls):
```

Comment:
@simahawk Why do you remove this way to create test models? What's the problem?

Reply:
@lmignon does not work on v11, it breaks transaction. In any case, my replacement sounded like less "black magic" and is working fine in a lot of modules ;)

Reply:
@simahawk It breaks transaction because one line is missing to disable the commit done by Odoo when a model is created. `self.env.cr.commit = mock.MagicMoxk()` or `self.env.cr.commit = lambda a: True` 
 With your code, after the tests the partner model is modified into database.

Reply:
what do you mean w/ modified? You run this only in test mode, so: db is a test one, everything is rolled back after tests (if you don't commit). Am I missing anything?